scripts/package.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+BLUE='\033[0;34m'
+RS='\033[0m' # reset
+
+echo -e "${RS}${BLUE} Cleaning and preparing..."
+find . -iname "*.tgz" -type f | grep -v 'docs/' | xargs rm
+
+echo -e "${RS}${BLUE} Linting and checking...${RS}"
+scripts/lint.sh
+RET=$?
+
+echo -e "${RS}${BLUE} Packaging charts... ${RS}"
+helm package .
+RET=$?
+
+if [[ "${RET}" == 0 ]]; then
+    helm repo index .
```

Comment:
errors from here isn't being caught

it might be easier to `set -ex` at the top instead of managing/checking exit codes for each command?

Reply:
yes, I think your suggesting is fine